docs/docs/training-data-format.mdx

```diff
@@ -685,3 +686,72 @@ rasa test
 
 If you want to know more about testing head over to
 [Testing Your Assistant](testing-your-assistant.mdx).
+
+
+## End-to-end Training
+
+:::caution experimental feature
+End-to-end training is an experimental feature.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+With end-to-end training, you do not have to deal with the specific
+intents of the messages that are extracted by the NLU pipeline.
+Instead, you can simply put the text of the user message directly in the stories,
+by using `user:` key.
+
+These end-to-end user messages follow the format:
+
+```yaml-rasa
+stories:
+- story: user message structure
+  steps:
+    - user: the actual text of the user message
```

Comment:
should there be an action after this ?

Reply:
yes but in the example for intents on lines 404-410 there is no actions

Reply:
I just copy pasted

Reply:
Ok, then how about adding an action here?

Reply:
done